cadquery/occ_impl/shapes.py

```diff
@@ -2558,7 +2558,6 @@ def fillet(
         """
         Apply 2D or 3D fillet to a wire
 
-        :param wire: The input wire to fillet. Currently only open wires are supported
         :param radius: the radius of the fillet, must be > zero
         :param vertices: Optional list of vertices to fillet. By default all vertices are fillet.
```

Comment:
```suggestion
        :param vertices: Optional list of vertices to fillet. By default all vertices are filleted.
```

Reply:
Thanks @nobkd .  I find the OCCT user guide phrases this as specifying vertices to be deleted.   In the context of creating a fillet on a planar face: 

"To create a fillet on planar face: define it, indicate, which vertex is to be deleted, and give the fillet radius ..."

I'll try an update along those lines.